src/plugins/flare/flareplugin.cpp

```diff
@@ -367,10 +394,14 @@ bool FlarePlugin::write(const Tiled::Map *map, const QString &fileName)
                     out << "," << w << "," << h << "\n";
 
                     // write all properties for this object
-                    Properties::const_iterator it = o->properties().constBegin();
-                    Properties::const_iterator it_end = o->properties().constEnd();
-                    for (; it != it_end; ++it) {
-                        out << it.key() << "=" << it.value().toString() << "\n";
+                    QVariantMap propsMap = o->properties();
+                    for (QVariantMap::const_iterator it = propsMap.constBegin(); it != propsMap.constEnd(); ++it) {
+                        if(it->userType() == filePathTypeId()){
```

Comment:
Sorry to be such a nitpick, but please also add the spaces here and adjust the `else` below as well.

Reply:
No worries. Just pushed a fix.